.github/workflows/build-gui-release-binaries.yml

```diff
@@ -227,8 +227,12 @@ jobs:
           ls -la /tmp/dmgs
           
           # Calculate SHA256 checksums
-          AARCH64_DMG=/tmp/dmgs/*_aarch64.dmg
-          X64_DMG=/tmp/dmgs/*_x64.dmg
+          AARCH64_DMG=$(ls /tmp/dmgs/*_aarch64.dmg)
```

Comment:
Please use the exact regex pattern used here: https://github.com/eigenwallet/core/blob/ed9122fd2a416c7679c9cc309ec4ad8291519643/dev-scripts/homebrew/eigenwallet.rb.template#L4-L12